lib/u3d.rb

```diff
@@ -48,4 +48,15 @@
 module U3d
   Helper = U3dCore::Helper
   UI = U3dCore::UI
+
+  def self.const_missing(const_name)
+    deprecated = {
+      PlaybackEngineUtils: IvyPlaybackEngineUtils,
+      INIParser: INIModulesParser
+    }
```

Comment:
I would move this out of the method, maybe to a constant like DEPRECATED_CLASSES or something

Reply:
Will do in a future cleanup.